src/main/java/bank/BankSystem.java

```diff
@@ -0,0 +1,68 @@
+package bank;
+
+import account.Account;
+import exception.system.AccountStatusException;
+import exception.account.BelowTargetException;
+import exception.account.InsufficienBalancetException;
+import exception.system.NotFoundAccountException;
+import lombok.RequiredArgsConstructor;
+import validate.ValidationUtils;
+
+import java.math.BigDecimal;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+
+@RequiredArgsConstructor
+public class BankSystem {
+
+    private static final String ACCOUNT_INACTIVE_ERROR = "비활성화 계좌입니다.";
+
+    private final BankStorage bankStorage;
+
+    public String createAccount(Account account) {
+        String accountNum = createAccountNum();
+        account.setAccountNum(accountNum);
+        return bankStorage.save(account);
+    }
+
+    public String deposit(String accountNum, BigDecimal balance) throws AccountStatusException, NotFoundAccountException {
+        Account account = ValidationUtils.getAccount(getAllList(), accountNum);
+        if (ValidationUtils.validateStatus(account)) return account.deposit(balance);
```

Comment:
```suggestion
        if (ValidationUtils.validateStatus(account)) {
            return account.deposit(balance);
        }
```
한 줄이어도 괄호로 감싸주는 것이 가독성에 좋습니다